templates/card.html

```diff
@@ -0,0 +1,48 @@
+<!-- CARD -->
+<div class="card">
+    {% if remove_enabled %}
+      <button>
+        <i class="fas fa-trash"></i>
+      </button>
+    {% endif %}
+    <div>
+      <section class="left-part">
+        {% if service.logo_url %}
+        <img src="{{ service.logo_url }}" title="{{ service.name }} Logo" />
+        {% else %}
+        <img src="{{ static('img/no_logo.png') }}" title="Logo Unavailable" />
+        {% endif %}
+        <ul>
+          <li>
+            <a href="{{ service.visit_us }}"><i class="fas fa-globe"></i> Visit us</a>
+          </li>
+          <li>
+            <a href="{{ service.beacon_api }}"><i class="fas fa-share-square"></i>Beacon API</a>
```

Comment:
wrap them in if-then-else, in case we don't have the values

Reply:
added the if statements each value